src/super_gradients/training/sg_trainer/sg_trainer.py

```diff
@@ -221,6 +214,73 @@ def train_from_config(cls, cfg: Union[DictConfig, dict]) -> None:
                       valid_loader=val_dataloader,
                       training_params=cfg.training_hyperparams)
 
+    @classmethod
+    def resume_experiment(cls, experiment_name: str, ckpt_root_dir: str = None) -> None:
```

Comment:
Maybe add some example how it can and should be used.

Reply:
updated